src/main/java/edu/kis/powp/jobs2d/drivers/SelectEventSubscriber.java

```diff
@@ -0,0 +1,10 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.powp.jobs2d.features.DriverFeature;
+import edu.kis.powp.observer.Subscriber;
+
+public class SelectEventSubscriber implements Subscriber {
```

Comment:
The name of the subscriber doesn't explain anything at all.
What does the subsriber do?
Optionally: What kind of event are we meant to be subscribed to?

Reply:
After a little brainstorming we both agreed that "DriverChangeEventSubscriber" will be a better name for this class